modules/site-health/audit-autoloaded-options/load.php

```diff
@@ -117,3 +117,44 @@ function perflab_aao_autoloaded_options_size() {
 	global $wpdb;
 	return (int) $wpdb->get_var( 'SELECT SUM(LENGTH(option_value)) FROM ' . $wpdb->prefix . 'options WHERE autoload = \'yes\'' );
 }
+
+/**
+ * Fetches autoload top list.
+ *
+ * @since n.e.x.t
+ *
+ * @return array Autoloaded data as option names and their sizes.
+ */
+function perflab_aao_query_autoloaded_options() {
+	global $wpdb;
+
+	$option_threshold = apply_filters( 'perflab_aao_autoloaded_options_table_threshold', 100 );
+
+	var_dump( $wpdb->options );
+
+	return $wpdb->get_results( $wpdb->prepare( "SELECT option_name, LENGTH(option_value) AS option_value_length FROM {$wpdb->options} WHERE autoload='yes' AND LENGTH(option_value) > %d ORDER BY option_value_length DESC LIMIT 20", $option_threshold ) );
+}
+
+/**
+ * Gets formatted autoload options table.
+ *
+ * @since n.e.x.t
+ *
+ * @return string HTML formatted table.
+ */
+function perflab_aao_get_autoloaded_options_table() {
+	$autoload_summary = perflab_aao_query_autoloaded_options();
+
+	$html_table = sprintf(
+		'<table class="widefat striped"><thead><tr><th scope="col">%s</th><th scope="col">%s</th></tr></thead><tbody>',
+		esc_html__( 'Option Name', 'performance-lab' ),
+		esc_html__( 'Size (bytes)', 'performance-lab' )
+	);
+
+	foreach ( $autoload_summary as $value ) {
+		$html_table .= sprintf( '<tr><td>%s</td><td>%s</td></tr>', $value->option_name, $value->option_value_length );
```

Comment:
I think formatting the bytes to a human readable format would be good. Nobody knows what `1234567` bytes means.

Reply:
Agreed. We have to use `size_format( $value->option_value_length, 2 )` to show the value in human readable format.

Reply:
@rytisder Not sure if you saw this feedback. If you could follow up on that, it would be great. Thank you!

Reply:
:wave: @rytisder Checking in on the above comment, can you please follow up? Thanks!

Reply:
The `(bytes)` can be removed if `size_format()` is used.